serve.go

```diff
@@ -37,6 +37,8 @@ func Serve(handlers *types.FaaSHandlers, config *types.FaaSConfig) {
 	r.HandleFunc("/function/{name:[-a-zA-Z_0-9]+}", handlers.FunctionProxy)
 	r.HandleFunc("/function/{name:[-a-zA-Z_0-9]+}/", handlers.FunctionProxy)
 
+	r.HandleFunc("/system/id", handlers.IdentityHandler).Methods("GET")
```

Comment:
Maybe `/info` would suit this better and be less ambiguous than `identity`? @stefanprodan